### PR TITLE
test: add v1.15.0-beta.0 to the CI

### PR DESCRIPTION
**Pull request by aanm**
Signed-off-by: André Martins <andre@cilium.io>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/8050)

### CODE REVIEW

**Comment by aanm** [comment]
test-missed-k8s

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]

**Comment by ianvernon** [comment]
@aanm why does this need backporting?

**Comment by aanm** [comment]
> @aanm why does this need backporting?

We decided that we need to support new releases of k8s versions on already cilium releases.

File: ginkgo-kubernetes-all.Jenkinsfile [hunk 1]
```diff
@@ -108,15 +108,16 @@ pipeline {
             }
         }
 
-        stage('Boot VMs 1.13'){
+        stage('Boot VMs 1.{13,15}'){
```
Comment (nebril) [inline ginkgo-kubernetes-all.Jenkinsfile:111]: Can you put provisioning vms and running tests in parallel stages with separate directories (same way as for 1.11 and 1.12 above)?

**Review by nebril — CHANGES_REQUESTED** [review]
Overall looks good, but it will run 1.15 tests in parallel to 1.13 tests in the same directory which can lead to cross polution between tests.

**Review by nebril — APPROVED** [review]

File: ginkgo-kubernetes-all.Jenkinsfile (hunk 1 above)
Comment (nebril) [reply ginkgo-kubernetes-all.Jenkinsfile:111]: My bad, I thought my PR got merged before.

**Pull request merged**